recipes/lintegrate/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "lintegrate" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 57b7caad93c87cac87b77af983a0167fddfd3011819537c1148c0ea35ec02c01
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - gsl
+    - numpy
+    - python
+    - pip
+  run:
+    - gsl
+    - numpy
+    - python
+
+test:
+  imports:
+    - lintegrate
+
+about:
+  home: http://github.com/mattpitkin/lintegrate
+  license: GPLv3
+  license_family: GPL
+  license_file: LICENSE.txt
```

Looking at the manifest it looks like the licence is already in the tarball
```suggestion
  license_file: LICENSE
```